rows_test.go

```diff
@@ -204,19 +204,42 @@ func TestReadingFileContents(t *testing.T) {
 
 	require.Equal(t, []Row{
 		{Index: 1, Cells: []Cell{
-			{Column: "A", Row: 1, Value: "rec_id"},
-			{Column: "B", Row: 1, Value: "culture"},
-			{Column: "C", Row: 1, Value: "sex"},
+			{Column: "A", Index: 0, Row: 1, Value: "rec_id"},
+			{Column: "B", Index: 1, Row: 1, Value: "culture"},
+			{Column: "C", Index: 2, Row: 1, Value: "sex"},
 		}},
 		{Index: 2, Cells: []Cell{
-			{Column: "A", Row: 2, Value: "rec-67374-org"},
-			{Column: "B", Row: 2, Value: "usa"},
-			{Column: "C", Row: 2, Value: "f"},
+			{Column: "A", Index: 0, Row: 2, Value: "rec-67374-org"},
+			{Column: "B", Index: 1, Row: 2, Value: "usa"},
+			{Column: "C", Index: 2, Row: 2, Value: "f"},
 		}},
 		{Index: 3, Cells: []Cell{
-			{Column: "A", Row: 3, Value: "rec-171273-org"},
-			{Column: "B", Row: 3, Value: "ara"},
-			{Column: "C", Row: 3, Value: "m"},
+			{Column: "A", Index: 0, Row: 3, Value: "rec-171273-org"},
+			{Column: "B", Index: 1, Row: 3, Value: "ara"},
+			{Column: "C", Index: 2, Row: 3, Value: "m"},
 		}},
 	}, rows)
 }
+
+func TestColumnRefs(t *testing.T) {
+	test := func(s string, n int) {
+		t.Helper()
+		require.Equal(t, s, asCol(n)) // test the test
+		require.Equal(t, n, asIndex(s))
+	}
+	test("A", 0)
```

Comment:
This would be preferred as a Table Driven Test (https://github.com/golang/go/wiki/TableDrivenTests), rather than one test with lots of lines.